README.md

```diff
@@ -68,8 +68,22 @@ zfs | Exposes [ZFS](http://open-zfs.org/) performance statistics. | [Linux](http
 
 ### Disabled by default
 
-The perf collector may not work by default on all Linux systems due to kernel
-configuration and security settings. To allow access, set the following sysctl
+`node_exporter` also implements a number of collectors that are disabled by default.  Reasons for this vary by
+collector, and may include:
+* High cardinality
+* Prolonged runtime that exceeds Prometheus` `scrape_interval` or `scrape_timeout`
+* Significant resource demands on the host
+
+You can enable additional collectors as desired by adding them to your init system's service description for `node_exporter`
```

Comment:
How about "service supervisor", as this covers things like Docker and other non-init supervisors like runit.

```suggestion
You can enable additional collectors as desired by adding them to your service supervisor configuration for the `node_exporter`
```

Reply:
Hrm, hadn't thought of that.  I pushed a hybrid wording, PTAL.